podcasts/EpisodeDetailViewController+Actions.swift

```diff
@@ -51,10 +51,20 @@ extension EpisodeDetailViewController {
             if !PlaybackManager.shared.playing() {
                 dismiss(animated: true, completion: nil)
             }
+            if let timestamp = timestamp {
+                DataManager.sharedManager.saveEpisode(playedUpTo: timestamp, episode: episode, updateSyncFlag: true)
+                PlaybackManager.shared.seekTo(time: timestamp, startPlaybackAfterSeek: false)
+                updateProgress()
+            }
 
             PlaybackActionHelper.playPause()
         } else {
             dismiss(animated: true, completion: nil)
+            if let timestamp = timestamp {
+                DataManager.sharedManager.saveEpisode(playedUpTo: timestamp, episode: episode, updateSyncFlag: true)
+                DataManager.sharedManager.saveEpisode(playingStatus: .inProgress, episode: episode, updateSyncFlag: true)
```

Comment:
Can't we do

```suggestion
                episode.playingStatus = PlayingStatus.inProgress.rawValue
                episode.playedUpTo = timestamp
                DataManager.sharedManager.save(episode: episode)
```

This way we avoid doing two subsequent database calls changing the same episode.

Also, why do we need the `updateSyncFlag` to be `true`?

Reply:
I don't think we need it.

Reply:
@leandroalonso I changed to false and all is working correctly, I also followed your other suggestion and all is ready for another round of review.